scss/_custom-forms.scss

```diff
@@ -143,10 +143,17 @@
   -moz-appearance: none;
   -webkit-appearance: none;
 
+  @if $enable-rounded {
+    @include border-radius($input-border-radius);
```

Comment:
This is semantically wrong because the `border-radius` mixin include the `@if`/`@else` block already:

```scss
@mixin border-radius($radius: $border-radius) {
  @if $enable-rounded {
    border-radius: $radius;
  }
}
```

You'd probably want the inside of that mixin along here—just the `border-radius` property.